be/src/storage/lake/tablet_manager.cpp

```diff
@@ -335,6 +345,7 @@ Status TabletManager::put_aggregate_tablet_metadata(std::map<int64_t, TabletMeta
     put_fixed64_le(&fixed_buf, serialized_buf.size());
     RETURN_IF_ERROR(meta_file->append(Slice(fixed_buf)));
     RETURN_IF_ERROR(meta_file->close());
+    _metacache->cache_aggregation_partition(partition_location, true);
```

Comment:
Do we need to cache whole aggregate meta? Or cache each tablet's meta when publish is enough?

Reply:
> Do we need to cache whole aggregate meta? Or cache each tablet's meta when publish is enough?

We don't cache the aggregate meta, just cache the partition bucket path and mark this path, and subsequent queries will prioritize reading metadata using the ​​aggregated meta​​ approach.